README.md

```diff
@@ -40,34 +79,55 @@ See a complete working example in the [roots-example-project.com repo](https://g
   * `WP_HOME` - Full URL to WordPress home (http://example.com)
   * `WP_SITEURL` - Full URL to WordPress including subdirectory (http://example.com/wp)
   * `AUTH_KEY`, `SECURE_AUTH_KEY`, `LOGGED_IN_KEY`, `NONCE_KEY`, `AUTH_SALT`, `SECURE_AUTH_SALT`, `LOGGED_IN_SALT`, `NONCE_SALT` - Generate with [wp-cli-dotenv-command](https://github.com/aaemnnosttv/wp-cli-dotenv-command) or from the [WordPress Salt Generator](https://api.wordpress.org/secret-key/1.1/salt/)
-4. Add theme(s) in `web/app/themes` as you would for a normal WordPress site.
-5. Set your site vhost document root to `/path/to/site/web/` (`/path/to/site/current/web/` if using deploys)
-6. Access WP admin at `http://example.com/wp/wp-admin`
 
-## Deploys
+### 4. Configure your virtual host
+
+Set your site vhost document root to `/path/to/site/web/`. For instance, `/Users/John/cnhcirclek.org/web`.
+
+### 5. Build the theme
+
+Go to the CNH theme directory.
+
+**Example:**
+```
+cd web/app/themes/cnh
+```
+
+Build the assets.
+
+```
+gulp build
+```
+
+> Gulp is a task runner that helps us compile and build the assets for the website.
+
+> The website needs to be built because the styles are actually written in [SCSS](https://sass-lang.com)
+
+Refer to the [Frontend Development](#frontend-development) section for more commands.
 
-There are two methods to deploy Bedrock sites out of the box:
+### 6. Access admin panel
 
-* [Trellis](https://github.com/roots/trellis)
-* [bedrock-capistrano](https://github.com/roots/bedrock-capistrano)
+Access the admin panel at `http://example.com/wp/wp-admin`
 
-Any other deployment method can be used as well with one requirement:
+### 7. Set up pages and other content
 
-`composer install` must be run as part of the deploy process.
+Some of the pages on the website are defined using page templates in the [CNH theme](https://github.com/CNHCircleK/cnhcirclek.org/tree/master/web/app/themes/cnh). You'll have to create pages with specific slugs to access them and develop, if necessary.
 
-## Documentation
+Pages:
```

Comment:
I don't remember the knowdboard page but for DCON, I used a template because it was easier to customize the layout and the styling for DCON. DCON wanted to have its own style but I wanted the page to still be a part of the website. Other pages can conform to the templates that already exist.